centreon/www/front_src/src/AgentConfiguration/Listing/DeleteModal.tsx

```diff
@@ -54,21 +56,11 @@ const DeleteModal = (): JSX.Element => {
       </Modal.Header>
       <Modal.Body>
         <Typography>
-          {hasPoller ? (
-            <Trans t={t}>
-              You are going to delete the configuration for the{' '}
-              <strong>{{ poller }}</strong> poller from the{' '}
-              <strong>{{ agent }}</strong> agent configuration. All
-              configuration parameters for this poller will be deleted. This
-              action cannot be undone.
-            </Trans>
-          ) : (
-            <Trans t={t}>
-              You are going to delete the <strong>{{ agent }}</strong> agent
-              configuration. All configuration parameters for this agent will be
-              deleted. This action cannot be undone.
-            </Trans>
-          )}
+          {sanitizedHTML({
+            initialContent: hasPoller
+              ? t(labelDeletePollerConfirmation, { poller, agent })
+              : t(labelDeleteAgentConfirmation, { agent })
+          })}
```

Comment:
Use the Trans component instead: https://react.i18next.com/latest/trans-component
Something like this
```
<Trans
  i18nKey="myKey" // optional -> fallbacks to defaults if not provided
  defaults="hello <italic>beautiful</italic> <bold>{{what}}</bold>" // optional defaultValue
  values={{ what: 'world'}}
  components={{ italic: <i />, bold: <strong /> }}
/>
```

Reply:
Nope, the translation doesn't work with this implementation, I fixed it for you